internal/tools/gittar/metrics/collector.go

```diff
@@ -165,20 +166,39 @@ func (c *Collector) IterateRepos(req apistructs.GittarListRepoRequest) ([]*Perso
 		gitRepository.Url = conf.GittarUrl() + "/" + repo.Path
 		gitRepository.IsExternal = repo.IsExternal
 
-		branch, err := gitRepository.GetDefaultBranch()
+		branches, err := gitRepository.GetBranches()
 		if err != nil {
-			logrus.Warningf("failed to get default branch for repo: %s, err: %v, use branch: %s as default", repo.Path, err, defaultBranch)
-			branch = defaultBranch
+			logrus.Warningf("failed to get all branch for repo: %s, err: %v, use branch: %s as default", repo.Path, err, defaultBranch)
+			branches = append(branches, defaultBranch)
```

Comment:
defaultBranch may be duplicated in branches

Reply:
ok,thanks